src/Http/Cookie/CookieCollection.php

```diff
@@ -177,18 +177,25 @@ public function getIterator()
      * when this method is called will be applied to the request.
      *
      * @param \Psr\Http\Message\RequestInterface $request The request to update.
+     * @param array $extraCookies Associative array of additional cookies to add into the request. This
+     *   is useful when you have cookie data from outside the collection you want to send.
      * @return \Psr\Http\Message\RequestInterface An updated request.
      */
-    public function addToRequest(RequestInterface $request)
+    public function addToRequest(RequestInterface $request, array $extraCookies = [])
```

Comment:
we can not send object  `new Cookie('api', 'A', null, '/api', 'example.com'))` to $extraCookies
that is true?